drivers/built_for_zero_report/app/views/built_for_zero_report/warehouse_reports/bfz/_filter.haml

```diff
@@ -0,0 +1,14 @@
+= simple_form_for :report, url: built_for_zero_report_warehouse_reports_bfz_index_path, method: :get do |f|
+  - content_for :filters_col_full do
+    .row
+      .col-sm-3
+        = f.input :start_date, as: :date_picker, input_html: { value: @start_date }
+      .col-sm-3
+        = f.input :end_date, as: :date_picker, input_html: { value: @end_date }
+      .col-sm-6
+        = f.input :section, as: :select_two, collection: BuiltForZeroReport.sections, selected: @section_key
```

Comment:
Needs to be limited to enabled System Cohorts and should throw a big warning if you don't have any enabled with a link to site config if you can get there.

Reply:
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/33072305/166007625-9d2e684a-7177-4f7d-a41e-3c9cd3620f37.png">